scripts/build.sh

```diff
@@ -55,7 +55,7 @@ function run::build() {
 
       if command -v upx > /dev/null; then
         printf "%s" "compressing... "
-        upx -q -9 run > /dev/null
+        upx -q -9 --exact run > /dev/null
```

Comment:
Wouldn't this be overwritten by github-config?

Reply:
yes. I am just using this PR to debug what is happening. I think the issue is with UPX, but I cannot reproduce it anywhere other than in a PR on this repo.